### ISSUE TITLE
oAuth is flat 500ing

### ISSUE THREAD

**Issue by BarryCarlyon** [twitchdev/issues#685]
**Brief description**

In some cases people are not completing oAuth

Either outbound query string is being stripped resuling in 400 missing client ID
or
after clicking accept the user is being redirected to `https://id.twitch.tv/oauth2/'`

**How to reproduce**

It's unclear how to reliably replicate as it doesn't affect me.

**Expected behavior**

oAuth to work

**Comment by BarryCarlyon** [comment]
Additional forum report https://discuss.dev.twitch.tv/t/implicit-grant-flow-oauth-broken-for-some-users/41651

And a couple of Discord reports

**Comment by Xemdo** [comment]
This should be working now. If the issue persists for anyone, they should clear their cache/cookies relating to Twitch, or try in Incognito/Private Browsing.

**Comment by penance1** [comment]
> This should be working now. If the issue persists for anyone, they should clear their cache/cookies relating to Twitch, or try in Incognito/Private Browsing.

Neither suggested solution helped to resolved this issue.

**Comment by AlphaDuplo** [comment]
Can confirm, the issue is back for about an hour now. It has been working before though.

**Comment by jbulava** [comment]
@AlphaDuplo Can you share what apps/client_ids of yours are experiencing the issue? Feel free to message me directly with that info.

**Comment by AlphaDuplo** [comment]
The issue seems to be fixed now. We're seeing new users authenticating successfully again.

@jbulava I'll DM you anyways just in case there's anything to investigate though :)

**Comment by penance1** [comment]
Battle.net appears to be properly authenticating again.

**Comment by BarryCarlyon** [comment]
![image](https://user-images.githubusercontent.com/20999/200067272-369aaa8f-c59a-42f6-9898-24b9eb6f6806.png)
it's flat 500-ing right now

**Comment by KatarinaWyatt** [comment]
We're getting 500 errors on our Oauth callback in both Chrome and FF when trying to login via Twitch on our website , I've not made any changes to the script in question for at least a couple of months and it was working seamlessly for more than 2 years before that. We've tried logging in with 3 different accounts we've got access to, clearing caches, restarting machines, all the usual stuff but to no avail. Our entire streaming setup requires this to work so I'd be extremely grateful of a solution - preferably a robust one which doesn't force our users to have to do anything remotely complicated or technical because some of them are very un-tech savvy.
![500error](https://user-images.githubusercontent.com/68875393/200073015-6c6c4624-26a4-4d44-9cbf-6e5b4eef049a.jpg)
